backend/Services/SightingService.cs

```diff
@@ -51,8 +54,21 @@ public Sighting CreateSighting(CreateSightingRequest request)
                 Longitude = request.Longitude,
                 ConfirmationStatus = ConfirmationStatus.Pending,
             };
-
             return _sightings.CreateSighting(newSighting);
         }
+
+        public Sighting ConfirmOrRejectSighting(ConfirmOrRejectRequest confirmOrRejectSighting, int sightingId)
+        {
+            if (confirmOrRejectSighting.NewConfirmationStatus == ConfirmationStatus.Approved)
+            {
+                return _sightings.ConfirmRequest(sightingId);
+            }
+            else if (confirmOrRejectSighting.NewConfirmationStatus == ConfirmationStatus.Rejected)
+            {
+                return _sightings.RejectRequest(sightingId);
+            }
+            throw new ArgumentOutOfRangeException("The confirmation request was not to approve or reject the sighting." +
+                "To approve, the NewConfirmationStatus of the request should be 2. To reject, it should be 1.");
```

Comment:
This is really nice, but you can (read: should!) be extra secure by grabbing the exact integer value of the specific status:
```suggestion
                $"To approve, the NewConfirmationStatus of the request should be {(int)ConfirmationStatus.Approved}. To reject, it should be {(int)ConfirmationStatus.Rejected}.");
```
The brackets with a type inside, like `(int)`, mean "explicitly cast this value to another type". It's _not_ the same as "parsing", which is usually converting a _string_ to another value - bracket casting is for types that are actually properly interchangeable (like `int` and `uint`, or `float` and `double`) and should be used sparingly. Here it's alright though!